vllm/_custom_ops.py

```diff
@@ -707,6 +707,36 @@ def scaled_fp8_quant(
     return output, scale
 
 
+# semi structured fp8
+def semi_structured_fp8_compress(input: torch.Tensor) -> torch.Tensor:
+    assert input.dtype == torch.float8_e4m3fn
+    return torch.ops._C.cslt_compress_fp8_semi_structured(input)
+
+
+def semi_structured_mm(
+        A_compressed: torch.Tensor,
+        B_dense: torch.Tensor,
+        scale: Optional[float] = None,
+        bias: Optional[torch.Tensor] = None,
+        out_dtype: Optional[torch.dtype] = None) -> torch.Tensor:
+    return torch.ops._C.cslt_mm_semi_structured(A_compressed, B_dense, scale,
+                                                bias, out_dtype)
+
+
+def semi_structured_mm2(
+        A_compressed: torch.Tensor,
+        B_dense: torch.Tensor,
+        scale: Optional[float] = None,
+        bias: Optional[torch.Tensor] = None,
+        out_dtype: Optional[torch.dtype] = None) -> torch.Tensor:
+    return torch.ops._C.cslt_mm_semi_structured2(A_compressed, B_dense, scale,
+                                                 bias, out_dtype)
```

Comment:
Please make sure to add fake/meta functions for these ops (see `@register_fake` in this file for examples).  Some `opcheck` tests would be good as well.  See `tests/kernels/test_marlin_gemm.py` for an example of an opcheck test.

Let me know if you need any help with either of these.